tests/phpunit/tests/rest-api/rest-widget-types-controller.php

```diff
@@ -125,6 +125,39 @@ public function test_get_items() {
 		}
 	}
 
+	/**
+	 * @ticket 53303
+	 */
+	public function test_get_items_ordering() {
+		wp_set_current_user( self::$admin_id );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/widget-types' );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+		$this->assertGreaterThan( 1, count( $data ) );
+		$ids      = wp_list_pluck( $data, 'id' );
+		$expected = array(
+			'archives',
+			'block',
+			'calendar',
+			'categories',
+			'custom_html',
+			'media_audio',
+			'media_gallery',
+			'media_image',
+			'media_video',
+			'meta',
+			'nav_menu',
+			'pages',
+			'recent-comments',
+			'recent-posts',
+			'rss',
+			'search',
+			'tag_cloud',
+			'text',
+		);
+		$this->assertEqualSets( $expected, $ids );
```

Comment:
This test feels very fragile, it will require updating whenever we introduce or remove a new widget. Can we instead assert that the response is sorted?

Reply:
>Can we instead assert that the response is sorted? 
@TimothyBJacobs How would we do this?

Reply:
Something along the lines of this:

```php
$sorted = $ids;
sort( $sorted );

$this->assertSame( $sorted, $ids );
```

Reply:
I personally don't like tests like this, as it feels a little like 1 === 1. But if you are happy with it Tim, I will defer to your judgement.